### DISCUSSION TITLE
[Feature Request] Nested drop-down menu

### DISCUSSION THREAD

**Discussion by dt50** [heroui-inc/heroui#1895] [Feature requests]
### Is your feature request related to a problem? Please describe.

I need to use nested drop-down menu in dropdown.

### Describe the solution you'd like

1. Create Dropdown (with trigger and menu)
2. In DropdownItem create second Dropdown
3. In the browser it would look like a drop-down list in a drop-down list

### Describe alternatives you've considered

In shadcn/ui with Radix UI this "Menubar" is already realized
Link for [Menubar](https://ui.shadcn.com/docs/components/menubar)

### Screenshots or Videos

![CleanShot 2023-11-06 at 11 39 37@2x](https://github.com/nextui-org/nextui/assets/32037208/691274b8-08b0-40a6-970a-978da5723e95)

**Comment by aliamiri1380** [comment]
I've provided such functionality inside this repo:
https://github.com/aliamiri1380/fast-heroui


https://github.com/user-attachments/assets/78e666e0-817e-4689-9738-a4a2bef47288

full code:

```js
import  Dropdown  from  './parts/hero_dropdown'

<Dropdown subPlacement={'right'} items={[
		{ 'type': 'item', 'content': 'New file', 'icon': 'file-plus' },
		{
			'type': 'sub', 'content': 'Copy link', 'icon': 'link', 'items': [
				{ 'type': 'item', 'content': 'Copy full link', 'icon': 'link-plus' },
				{ 'type': 'item', 'content': 'Copy short link', 'icon': 'link-plus', 'iconProps': { 'size': 'md' } },
				{
					'type': 'sub', 'content': 'Copy colorful link', 'icon': 'link-plus', 'items': [
						{ 'type': 'item', 'content': 'red', 'props': { 'color': 'danger' } },
						{ 'type': 'item', 'content': 'green', 'props': { 'color': 'success' } },
						{ 'type': 'item', 'content': 'blue', 'props': { 'color': 'primary' } },
					]
				},
			]
		},
		{ 'type': 'item', 'content': 'Edit file', 'icon': 'pencil' },
		{ 'type': 'item', 'content': 'Delete file', 'icon': 'trash', 'props': { 'color': 'danger' } },
	]}>
		<Button>Open Menu</Button>
	</Dropdown>
```

see the readme for more details

**Reply by melodyxpot** [reply]
Wow, I love it.
Can we implement this feature into HeroUI

**Reply by aliamiri1380** [reply]
surely, I hope see such things more
